app/models/data-operation.ts

```diff
@@ -520,6 +532,26 @@ export default class DataOperation {
     this.model.format.interpolation = interpolationMethod;
   }
 
+  /**
+   * Sets the spatial point to be used for spatial subsetting, an array of 2 coordinates:
+   *   [ Longitude, Latitude ]
+   *
+   * @param point - The spatial point in form of [ Longitude, Latitude ]
+   */
+  set spatialPoint(point: Array<number>) {
+    this.model.point = point;
```

Comment:
I'm thinking that point should be under subset (`this.model.subset.point`). That will affect the schema in addition to the model changes in this file.